client/extensions/woocommerce/app/order/order-refund-card.js

```diff
@@ -109,30 +128,41 @@ class OrderRefundCard extends Component {
 		const refundObj = {
 			amount: this.state.refundTotal + '', // API expects a string
 			reason: this.state.refundNote,
-			// api_refund
+			api_refund: ( -1 !== paymentMethod.method_supports.indexOf( 'refunds' ) ),
```

Comment:
It doesn't need done for this PR, but if we start using more method support checks in the future (besides the two in this file), a helper function that returns a bool would be nice.